tldr.py

```diff
@@ -162,6 +162,7 @@ def get_language_list():
         lambda x: x.split('_')[0],
         filter(lambda x: x != 'C', languages)
```

Comment:
we should be able to get the desired intent by changing this filter line to be `x not in ['C', '']`.